frontend/packages/console-shared/src/components/status/status.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
+import * as History from 'history';
+import { Link } from 'react-router-dom';
+import { Popover } from '@patternfly/react-core';
+import { Icon, Button } from 'patternfly-react';
+
+type StatusProps = {
+  icon?: string;
+  children: React.ReactNode;
+};
+
+export const Status = ({ icon, children }: StatusProps) => (
+  <>
+    {icon && <Icon type="pf" name={icon} className="kubevirt-status__icon" />} {children}
```

Comment:
Is this suppose to be limited to certain "status" icons? or can they be anything?
PF4 icons don't work this way as they are component based.

Reply:
`Icon` comes from `patternfly-react` package which means PF3 ([component showcase](https://patternfly-react.surge.sh/patternfly-3/index.html)).

Supported `type` includes `pf` (PatternFly specific) and `fa` (FontAwesome complementary). `name` depends on `type`.

In this case, since `type` is hardcoded to `pf`, `name` could be checked against a list of available PF icon names, but I'm not sure if such list exists.